griptape/artifacts/blob_artifact.py

```diff
@@ -1,26 +1,39 @@
 from __future__ import annotations
 
-import os.path
-from typing import Optional
+from typing import Any
 
 from attrs import define, field
 
 from griptape.artifacts import BaseArtifact
 
 
+def value_to_bytes(value: Any) -> bytes:
+    if isinstance(value, bytes):
+        return value
+    else:
+        return str(value).encode()
```

Comment:
should we move these converters to `utils`?

Reply:
I'm not sure, some of them do conversions very specific to the Artifact type. `CsvRowArtifact`, for instance.

I just updated the logic to live in the class, let me know what you think.

Reply:
the `CsvRowArtifact` one makes sense. im still not convinced that this one is class-specific enough. can we one-line it until it doesnt have to be?
```
converter=lambda x: x if isinstance(x, bytes) else str(x).encode()
```
?

Reply:
Sure, just updated.